group_vars/pulmap/production.yml

```diff
@@ -11,7 +11,8 @@ passenger_app_env: "production"
 passenger_server_name: "pulmap.princeton.edu"
 passenger_ruby: "/usr/local/bin/ruby"
 bundler_version: "2.3.18"
-desired_nodejs_version: "v16.15.0"
+nodejs_release_url_alt: "https://github.com/pulibrary/nodejs-binaries/releases/download/"
```

Comment:
I'd vote to use the existing variable instead of adding a separate variable and task for the alternative URL.

Do we want to put build Node binaries on our mirror instead of on GitHub?

Reply:
I wasn't able to get variable override to happen because in the [order of variable precedence](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#understanding-variable-precedence), task variables (like nodejs_release_url) are higher than variables defined in playbook var files. Now that I'm looking at it though, we might be able to move `nodejs_release_url` from `nodejs/vars/main.yml` to `nodejs/defaults/main.yml`

Reply:
As for the binaries on the mirror, how would we get them on to the server in an automated fashion? Is there an FTP service?

Reply:
Moving the `nodejs_release_url` variable didn't work. It still has a higher precedence.

Reply:
Fixed.